socorro/external/es/crashstorage.py

```diff
@@ -157,69 +260,10 @@ def save_raw_and_processed(self, raw_crash, dumps, processed_crash,
         except Exception:
             # NOTE(willkg): An error here shouldn't screw up saving data. Log it so we can fix it
             # later.
-            self.config.logger.exception('something went wrong when capturing raw_crash_size')
-
-        self.transaction(
-            self._submit_crash_to_elasticsearch,
-            crash_document=crash_document
-        )
-
-    @staticmethod
-    def reconstitute_datetimes(processed_crash):
-        datetime_fields = [
-            'submitted_timestamp',
-            'date_processed',
-            'client_crash_date',
-            'started_datetime',
-            'startedDateTime',
-            'completed_datetime',
-            'completeddatetime',
-        ]
-        for a_key in datetime_fields:
-            try:
-                processed_crash[a_key] = string_to_datetime(
-                    processed_crash[a_key]
-                )
-            except KeyError:
-                # not there? we don't care
-                pass
-
-    @staticmethod
-    def remove_bad_keys(data):
-        """Removes keys from the top-level of the dict that are bad
-
-        Good keys satisfy the following properties:
-
-        * have one or more characters
-        * are composed of a-zA-Z0-9_-
-
-        Anything else is a bad key and needs to be removed.
-
-        This modifies the data dict in-place and only looks at the top level.
-
-        :arg dict data: the data to remove bad keys from
-
-        """
-        if not data:
-            return
-
-        # Copy the list of things we're iterating over because we're mutating
-        # the dict in place.
-        for key in list(data.keys()):
-            if not is_valid_key(key):
-                del data[key]
+            self.config.logger.exception('something went wrong when capturing processed_crash_size')
```

Comment:
This fixes `raw` to `processed`. The original logging message was a copypasta fail.